app/listeners/hook_listener.rb

```diff
@@ -1,14 +1,14 @@
 class HookListener < BaseListener
   def message_created(event)
     message = extract_message_and_account(event)[0]
-    return unless message.webhook_sendable?
+    return unless message.slack_hook_sendable?
 
     execute_hooks(event, message)
   end
 
   def message_updated(event)
     message = extract_message_and_account(event)[0]
-    return unless message.webhook_sendable?
+    return unless message.slack_hook_sendable?
```

Comment:
There might be a better place to add this. This the underlying hooks vary.  There might be an integration which would want to listen to all message created/updated events. 

Let's remove this condition check from the listener and have the checks in individual integration processor services instead.

Reply:
Cool!